resources/public/api/conf/6.0/bsas_trigger.yaml

```
@@ -35,18 +35,30 @@ post:
         schema:
           $ref: "./schemas/trigger/request.json"
         examples:
-          example-1:
+          PRE-TY-2024-25-Self-Employment:
             description: Self-employment Trigger
             value:
-              $ref: "./examples/trigger/trigger_se_request.json"
-          example-2:
+              $ref: "./examples/trigger/def1/trigger_se_request.json"
+          TY-2024-25-Self-Employment:
+            description: Self-employment Trigger
```

```suggestion
            description: Self-employment Trigger for TY 2024-25 and after
```
likewise for the others...?